package.json

```diff
@@ -7,11 +7,12 @@
   "scripts": {
     "commit": "cz",
     "prepare": "husky install",
-    "build": "lerna run build --stream --parallel",
-    "test": "lerna run test --stream --parallel",
-    "clear": "lerna run clear --stream --parallel",
-    "release": "lerna run release --stream"
+    "build": "yarn workspaces foreach -pti --exclude @autoviews/docs run build",
+    "clear": "yarn workspaces foreach -pi run clear",
+    "release": "yarn workspaces foreach -i run release",
+    "test": "is-ci && jest --maxWorkers=7 || jest --maxWorkers=50%"
```

Comment:
`--maxWorkers=7` is requirement for internal Wix CI. Also, don't know if  any `--maxWorkers` affects jest performance on GitHub Actions